src/test/ui/const-generics/const-expression-parameter.rs

```diff
@@ -16,6 +15,8 @@ fn foo_b() {
 
 fn foo_c() {
     u32_identity::< -1 >(); // ok
+    // FIXME(const_generics)
```

Comment:
why `FIXME`? The error has nothing to do with const eval, the unary negation trait is not implemented for `u32`